.github/workflows/release-azure-pipelines.yml

```diff
@@ -93,40 +93,268 @@ extends:
+              targetType: inline
+              script: |
+                cd azureauth-${{ parameters.version }}-osx-arm64-signed/AzureAuth
+                chmod +x azureauth createdump *.dylib
+          - task: ArchiveFiles@2
+            displayName: Create osx-arm64 archive
+            inputs:
+              rootFolderOrFile: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-osx-arm64-signed/AzureAuth
+              includeRootFolder: false
+              archiveType: tar
+              tarCompression: gz
+              archiveFile: azureauth-${{ parameters.version }}-osx-arm64.tar.gz
 
-            - task: DotNetCoreCLI@2
-              displayName: Test
-              inputs:
-                command: test
-                arguments: --configuration release --no-restore
-            
-            - task: DotNetCoreCLI@2
-              displayName: Build artifacts
-              env:
-                ADO_TOKEN: $(System.AccessToken)
-              inputs:
-                command: publish
-                projects: src/AzureAuth/AzureAuth.csproj
-                arguments: -p:Version=${{ parameters.version }} --configuration release --self-contained true --runtime ${{ config.runtime }} --output dist/${{ config.runtime }}
-                publishWebProjects: false
-                zipAfterPublish: false
-                modifyOutputPath: true
\ No newline at end of file
+    - stage: release
+      displayName: Release
+      dependsOn: package
+      jobs:
+      - job: approval
+        displayName: Manual Approval
+        pool: server
+        timeoutInMinutes: 5760 # job times out in 4 days
+        steps:
+        - task: ManualValidation@0
+          timeoutInMinutes: 4320 # task times out in 3 days
+          inputs:
+            notifyUsers: $(REVIEWER)
+            instructions: 'Review the AzureAuth GitHub Release.'
+      - job: release
+        displayName: Release
+        dependsOn: approval
+        pool:
+          name: Azure-Pipelines-1ESPT-ExDShared
+          image: ubuntu-latest
+          os: linux
+        templateContext:
+          inputs:
+          - ${{ each config in parameters.buildConfigs }}:
+            - input: pipelineArtifact
+              artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
+              targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
+        steps:
+        - task: GitHubRelease@1
+          displayName: Create AzureAuth GitHub Release
+          inputs:
+            gitHubConnection: $(githubReleaseServiceConnection)
+            repositoryName: 'AzureAD/microsoft-authentication-cli'
+            action: 'create'
+            target: $(Build.SourceVersion)
+            tagSource: 'userSpecifiedTag'
+            tag: ${{ parameters.version }}
+            isPrerelease: ${{ parameters.prerelease }}
+            isDraft: true

```

Comment:
Are you going to update `isDraft` in further PRs?

Reply:
Correct, I will switch it to false when we use official template and switch pipeline to prod

Reply:
Created PR to this branch https://github.com/AzureAD/microsoft-authentication-cli/pull/394